mesa/experimental/altair_grid.py

```diff
@@ -0,0 +1,67 @@
+import json
+from typing import Callable
+
+import altair as alt
+import solara
+
+import mesa
+
+
+def get_agent_data_from_coord_iter(data):
```

Comment:
In my current implementation, I'm using an `agent_portrayal` method to generate the values needed to draw the space. That may be drawn from the old mesa visualization approach, IDK if there's a good way to pass in something like that to jupyterviz.

I mention because I wonder if it would be cleaner and more explicit than the way you're using json to dump and filter the agent dict.

Reply:
I agree with @rlskoeser; the JSON approach is not very clean. Is there a more explicit way to do this?